lib/test/iso/bmff/util/parseBox.ts

```diff
@@ -1,6 +1,6 @@
 import type { Box, BoxParser } from '@svta/common-media-library';
 import { parseFile } from './parseFile.ts';
 
-export function parseBox<T>(file: string, parser: BoxParser<T>, index: number): Box<T> {
+export function parseBox<T>(file: string, parser: BoxParser<T>, index: number): Box {
```

Comment:
Ideally, it has to return `T`, but also will require to cast return statement

```suggestion
export function parseBox<T extends Box>(file: string, parser: BoxParser<T>, index: number): T {
```